grails-app/views/aboutpage.gsp

```diff
@@ -1,130 +1,94 @@
-            <div class="landing-header" style="background-image: url('${resource(dir: "images/", file: "landing.jpg")}'); box-shadow:inset 0 0 0 2000px rgba(255,255,255,0.7);">
+            <div class="landing-header" style="background-image: url('${resource(dir: "images/", file: "bump.jpg")}'); box-shadow:inset 0 0 0 2000px rgba(210,105,30,0.5);">
                 <div class="container">
                     <div class="motto">
-                        <h1 class="title-uppercase"> About Groovy School</h1>
-                        <h3>Let us tell you more about what we do</h3>
+                        <h1 style="color:white;" >Hello,<br> We are Groovy School</h1>
+                        <h3 style="color:white;">Let us tell you more about what we do</h3>
                         </br>
                     </div>
                 </div>
             </div>
-            <div class="main">
-                <div class="section section-light-brown text-center landing-section">
-                    <div class="container">
-                        <div class="row">
-                            <div class="col-md-8 col-md-offset-2">
-                                <h2>What is Groovy School?</h2>
-                                <h5><br>   Groovy School is a series of free weekly training session/workshops where-in attendees are taught the Groovy programming language and some of the tools/frameworks within the Groovy ecosystem. The entire series lasts 6 months and is suitable for people who have never written any code before. At the end of the series, attendees are grouped and assigned an open source project to solve a real world problem affecting the community, using the knowledge and skills that they have obtained during the training.   </h5>
-                                </br>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-                <div class="section section-dark text-center landing-section">
-                    <div class="container">
-                        <h2>Meet our Instructors</h2>
+            <div class="section landing-section">
+                                <div class="container">
+                                <h3>What is Groovy School?</h3>
+                                <h5>Groovy School is a series of free weekly training session/workshops where-in attendees are taught the Groovy programming language and some of the tools/frameworks within the Groovy ecosystem. The entire series lasts 6 months and is suitable for people who have never written any code before. At the end of the series, attendees are grouped and assigned an open source project to solve a real world problem affecting the community, using the knowledge and skills that they have obtained during the training.</h5>
+                                <br />
+
+                                <h5>We <i class="fa fa-heart icon-danger"></i> &nbsp; what we do.</h5>
```

Comment:
please add random text for the what we do section, just to make it presentationally correct